modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/styles/components/_activation-keys.scss

```diff
@@ -41,11 +41,15 @@
 .cp-activation-key-icon {
 	border: 1px solid var(--color-neutral-2);
 	box-shadow: none;
-	left: 641px;
+	left: 648px !important;
 	min-width: fit-content;
-	top: 276px;
-	ul li span {
+	top: 276px !important;
```

Comment:
This is weird, why are you using static positioning here? I think that a better solution is to provide the parent element as a **position: relative** following the cp-activation-key-icon as a **position: absolute** and you can use the top and left props easily.

Reply:
Nice!